strr-web/stores/strr.ts

```diff
@@ -162,6 +193,29 @@ const primaryContact: PrimaryContactInformationI = {
   businessNumber: undefined
 }
 
+const propertyManager: PropertyManagerI = {
+  businessLegalName: '',
+  businessNumber: '',
+  businessMailingAddress: {
+    address: undefined,
```

Comment:
Why some empty and some undefined?

Reply:
required ones are undefined and optional are empty strings.

Reply:
Followed same pattern as the other code, primaryContact, etc.

Reply:
Fixed but this will likely need to be fixed in future for other code.

Reply:
Yea, the original code was not ideal.